### ISSUE TITLE
Crash on startup fabric 1.21.1

### ISSUE THREAD

**Issue by Elemen0** [CaelTheColher/Capes#148]
Every time I try to launch the game I get an immediate crash and this crash log:

```

A mod crashed on startup!
net.fabricmc.loader.impl.FormattedException: java.lang.RuntimeException: Could not execute entrypoint stage 'preLaunch' due to errors, provided by 'capes' at 'com.llamalad7.mixinextras.MixinExtrasBootstrap::init'!
	at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:63)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:162)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.RuntimeException: Could not execute entrypoint stage 'preLaunch' due to errors, provided by 'capes' at 'com.llamalad7.mixinextras.MixinExtrasBootstrap::init'!
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:403)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:401)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:160)
	... 2 more
Caused by: net.fabricmc.loader.api.EntrypointException: Exception while loading entries for entrypoint 'preLaunch' provided by 'capes'
	at net.fabricmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:56)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:399)
	... 3 more
Caused by: java.lang.RuntimeException: Mixin transformation of com.llamalad7.mixinextras.MixinExtrasBootstrap failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:427)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:534)
	at java.base/java.lang.Class.forName(Class.java:513)
	at net.fabricmc.loader.impl.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:50)
	at net.fabricmc.loader.impl.entrypoint.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:124)
	at net.fabricmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:53)
	... 4 more
Caused by: java.lang.ClassCircularityError: com/llamalad7/mixinextras/MixinExtrasBootstrap
	at knot//com.llamalad7.mixinextras.platform.fabric.MixinExtrasConfigPlugin.onLoad(MixinExtrasConfigPlugin.java:14)
	at org.spongepowered.asm.mixin.transformer.PluginHandle.onLoad(PluginHandle.java:119)
	at org.spongepowered.asm.mixin.transformer.MixinConfig.onSelect(MixinConfig.java:790)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.selectConfigs(MixinProcessor.java:498)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.select(MixinProcessor.java:460)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.checkSelect(MixinProcessor.java:438)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:290)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422)
	... 14 more

```

I have tried removing all of my other mods (sodium and wynntils) and I still get the same error message. Am I doing something wrong or missing anything?

**Comment by CaelTheColher** [comment]
can you send the full crash log file?

**Comment by CaelTheColher** [comment]
no response but I will assume this is the same error and fix as #149
